### PR TITLE
Add 9999th produce delay and records delay

### PR DESCRIPTION
**Pull request by xiowu0**
Add 9999th produce delay and records delay

### CODE REVIEW

File: src/main/java/com/linkedin/kmf/services/ProduceService.java [hunk 1]
```diff
@@ -251,7 +251,8 @@ public ProduceMetrics(final Metrics metrics, final Map<String, String> tags) {
       int sizeInBytes = 4 * bucketNum;
       _produceDelay.add(new Percentiles(sizeInBytes, _latencyPercentileMaxMs, Percentiles.BucketSizing.CONSTANT,
           new Percentile(new MetricName("produce-delay-ms-99th", METRIC_GROUP_NAME, "The 99th percentile delay in ms for produce request", tags), 99.0),
-          new Percentile(new MetricName("produce-delay-ms-999th", METRIC_GROUP_NAME, "The 999th percentile delay in ms for produce request", tags), 99.9)));
+          new Percentile(new MetricName("produce-delay-ms-999th", METRIC_GROUP_NAME, "The 999th percentile delay in ms for produce request", tags), 99.9),
```
Comment (abhishekmendhekar) [inline src/main/java/com/linkedin/kmf/services/ProduceService.java]: `The 999th percentile...` -> `The 99.9th percentile...`

File: src/main/java/com/linkedin/kmf/services/ProduceService.java [hunk 2]
```diff
@@ -251,7 +251,8 @@ public ProduceMetrics(final Metrics metrics, final Map<String, String> tags) {
...
+          new Percentile(new MetricName("produce-delay-ms-9999th", METRIC_GROUP_NAME, "The 9999th percentile delay in ms for produce request", tags), 99.99)));
```
Comment (abhishekmendhekar) [inline src/main/java/com/linkedin/kmf/services/ProduceService.java]: `The 9999th percentile...` -> `The 99.99th percentile...`

File: src/main/java/com/linkedin/kmf/services/ConsumeService.java [hunk 3]
```diff
@@ -250,7 +250,8 @@ public ConsumeMetrics(final Metrics metrics, final Map<String, String> tags) {
       int sizeInBytes = 4 * bucketNum;
       _recordsDelay.add(new Percentiles(sizeInBytes, _latencyPercentileMaxMs, Percentiles.BucketSizing.CONSTANT,
         new Percentile(new MetricName("records-delay-ms-99th", METRIC_GROUP_NAME, "The 99th percentile latency of records from producer to consumer", tags), 99.0),
-        new Percentile(new MetricName("records-delay-ms-999th", METRIC_GROUP_NAME, "The 999th percentile latency of records from producer to consumer", tags), 99.9)));
+        new Percentile(new MetricName("records-delay-ms-999th", METRIC_GROUP_NAME, "The 999th percentile latency of records from producer to consumer", tags), 99.9),
```
Comment (abhishekmendhekar) [inline src/main/java/com/linkedin/kmf/services/ConsumeService.java]: `The 999th percentile...` -> `The 99.9th percentile...`

File: src/main/java/com/linkedin/kmf/services/ConsumeService.java [hunk 4]
```diff
@@ -250,7 +250,8 @@ public ConsumeMetrics(final Metrics metrics, final Map<String, String> tags) {
...
+        new Percentile(new MetricName("records-delay-ms-9999th", METRIC_GROUP_NAME, "The 9999th percentile latency of records from producer to consumer", tags), 99.99)));
```
Comment (abhishekmendhekar) [inline src/main/java/com/linkedin/kmf/services/ConsumeService.java]: `The 9999th percentile...` -> `The 99.99th percentile...`

**Comment by d1egoaz** [comment]
@abhishekmendhekar  @lindong28 are there any plans to include this commit in the `2.0.0` tag? thanks!

https://github.com/linkedin/kafka-monitor/compare/2.0.0...master

**Comment by xiowu0** [comment]
> @abhishekmendhekar @lindong28 are there any plans to include this commit in the `2.0.0` tag? thanks!
> 
> [2.0.0...master](https://github.com/linkedin/kafka-monitor/compare/2.0.0...master)
> 
> 
> 
> or release maybe a newer version as this adds a new feature? `2.0.1`?

Done, created a 2.0.1 tag

**Comment by d1egoaz** [comment]
awesome, thanks!

**Pull request merged**